source/clear-linux/concepts/swupd-about.rst

```diff
@@ -14,6 +14,11 @@ Visit the `swupd man page`_ for more details.
 Versioning
 ==========
 
+While the process of describing installed software on other Linux\*
+distributions can be done with package managers, it can still be cumbersome to
+keep track of dissociated software version compatibility or to compare
+multiple systems. 
```

Comment:
Not quite what I meant. Fussy stuff. Recommended rev:
"It can be cumbersome to keep track of software version compatibility or compare multiple systems on many Linux distributions." [Then immediately after, comes Ln21]

Reply:
Oh, I now see I overlooked part of your previous suggestion. I think this line or something acknowledging the role/existence of package managers in other distros is important (and was raised as an argument):
> While the process of describing installed software on other Linux\*
distributions can be done with package managers

Reply:
Yes, I'm aware of that. I compressed that thought in my revised sentence, tacking on "
many Linux distributions" at the end.  My aim was to reduce the amount of times this document refers to "other Linux distributions." A short setup is all that's needed.